java/configuration/src/main/java/sleeper/configuration/properties/instance/CommonProperty.java

```diff
@@ -216,6 +216,18 @@ public interface CommonProperty {
             .validationPredicate(Utils::isTrueOrFalse)
             .propertyGroup(InstancePropertyGroup.COMMON)
             .build();
+    UserDefinedInstanceProperty TABLE_BATCHING_LAMBDAS_MEMORY_IN_MB = Index.propertyBuilder("sleeper.table.batching.lambdas.memory")
+            .description("The amount of memory for lambdas that create table batches.")
```

Comment:
Maybe "that create batches of tables to run some operation against, eg. create compaction jobs, garbage collection, partition splitting"?

Reply:
Thanks that sounds better, i've updated the description